network/discovery/metrics.go

```diff
@@ -27,6 +27,10 @@ var (
 		Name: "ssv:network:discovery:iterations",
 		Help: "Counts the number of times a node was iterated using the mixed iterator",
 	}, []string{"fork"})
+	metricSkippedPeers = promauto.NewCounterVec(prometheus.CounterOpts{
+		Name: "ssv:network:discovery:skipped_peers",
+		Help: "Counts skipped peers",
+	}, []string{"enr", "peer_id", "err"})
```

Comment:
Having peer ID or an ENR as a label is problematic because of its high cardinality (Prometheus struggles with this). We could use something such as `ssv:network:discovery:skipped_peers_total` for now if a total count would help. If not, we need some other solution.

Reply:
@zktaiga reverted the change and added the log rate limit